style.css

```diff
@@ -0,0 +1,29 @@
+.site {
+    font-family: 'Gotham Pro', sans-serif;
+    min-width: 320px;
+    max-width: 1170px;
+    overflow-x: hidden;
+    overflow-y: visible;
+    margin: 0 auto; 
+}
+.site a {
+    text-decoration: none;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/final-Axion/pullRequest?prid=5455661)